### PR TITLE
Remove jQuery - Part 1

### PR DESCRIPTION
**Pull request by MrTimscampi**
**Changes**

Replaces the following jQuery functions with native equivalents

* $.attr()
* $.addClass()
* $.removeClass()
* $.css()
* $.deferred
* $.each()
* $.filter()
* $.hide()
* $.html()
* $.is()
* $.parents()
* $.show()
* $.val()
* $.prop()
* $.trigger()

**Issues**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -43,7 +44,42 @@ module.exports = {
         'quotes': ['error', 'single', { 'avoidEscape': true, 'allowTemplateLiterals': false }],
         'semi': ['error'],
         'space-before-blocks': ['error'],
-        'space-infix-ops': 'error'
+        'space-infix-ops': 'error',
+        // jQuery Reemoval
```
Comment (barronpm) [inline .eslintrc.js:48]:
```suggestion
        // jQuery Removal
```

File: src/components/mediaLibraryCreator/mediaLibraryCreator.js [hunk 2]
```diff
@@ -52,9 +52,13 @@ define(['loading', 'dialogHelper', 'dom', 'jQuery', 'components/libraryoptionsed
     }
 
     function initEditor(page, collectionTypeOptions) {
-        $('#selectCollectionType', page).html(getCollectionTypeOptionsHtml(collectionTypeOptions)).val('').on('change', function () {
+        const selectCollectionType = $('#selectCollectionType', page);
```
Comment (ferferga) [inline src/components/mediaLibraryCreator/mediaLibraryCreator.js:55]: Why this one it's still using JQ?
```suggestion
        const selectCollectionType = page.querySelector('#selectCollectionType');
```

File: src/components/mediaLibraryCreator/mediaLibraryCreator.js [hunk 3]
```diff
@@ -68,11 +72,11 @@ define(['loading', 'dialogHelper', 'dom', 'jQuery', 'components/libraryoptionsed
 
                 if (index != -1) {
                     var name = this.options[index].innerHTML.replace('*', '').replace('&amp;', '&');
-                    $('#txtValue', dlg).val(name);
+                    dlg.querySelector('#txtValue').value = name;
                     var folderOption = collectionTypeOptions.filter(function (i) {
                         return i.value == value;
                     })[0];
-                    $('.collectionTypeFieldDescription', dlg).html(folderOption.message || '');
+                    $('.collectionTypeFieldDescription', dlg).innerHTML = folderOption.message || '';
```
Comment (ferferga) [inline src/components/mediaLibraryCreator/mediaLibraryCreator.js:79]: Same

File: src/components/tvproviders/schedulesdirect.js [hunk 4]
```diff
@@ -63,10 +63,12 @@ define(['jQuery', 'loading', 'globalize', 'emby-checkbox', 'listViewStyle', 'emb
 
                     return 0;
                 });
-                $('#selectCountry', page).html(countryList.map(function (c) {
+                const selectCountry = $('#selectCountry', page);
```
Comment (ferferga) [inline src/components/tvproviders/schedulesdirect.js]: Same

File: src/components/tvproviders/schedulesdirect.js [hunk 5]
```diff
@@ -186,7 +194,8 @@ define(['jQuery', 'loading', 'globalize', 'emby-checkbox', 'listViewStyle', 'emb
 
         function refreshListings(value) {
             if (!value) {
-                return void $('#selectListing', page).html('');
+                $('#selectListing', page).innerHtml = '';
```
Comment (ferferga) [inline src/components/tvproviders/schedulesdirect.js]: Same

File: src/components/tvproviders/schedulesdirect.js [hunk 6]
```diff
@@ -195,16 +204,16 @@ define(['jQuery', 'loading', 'globalize', 'emby-checkbox', 'listViewStyle', 'emb
                 url: ApiClient.getUrl('LiveTv/ListingProviders/Lineups', {
                     Id: providerId,
                     Location: value,
-                    Country: $('#selectCountry', page).val()
+                    Country: page.querySelector('#selectCountry').value
                 }),
                 dataType: 'json'
             }).then(function (result) {
-                $('#selectListing', page).html(result.map(function (o) {
+                $('#selectListing', page).innerHtml = result.map(function (o) {
```
Comment (ferferga) [inline src/components/tvproviders/schedulesdirect.js]: Same

File: src/components/tvproviders/schedulesdirect.js [hunk 7]
```diff
@@ -290,7 +299,7 @@ define(['jQuery', 'loading', 'globalize', 'emby-checkbox', 'listViewStyle', 'emb
                     page.querySelector('.selectTunersSection').classList.remove('hide');
                 }
             });
-            $('.createAccountHelp', page).html(globalize.translate('MessageCreateAccountAt', '<a is="emby-linkbutton" class="button-link" href="http://www.schedulesdirect.org" target="_blank">http://www.schedulesdirect.org</a>'));
+            $('.createAccountHelp', page).innerHtml = globalize.translate('MessageCreateAccountAt', '<a is="emby-linkbutton" class="button-link" href="http://www.schedulesdirect.org" target="_blank">http://www.schedulesdirect.org</a>');
```
Comment (ferferga) [inline src/components/tvproviders/schedulesdirect.js]: Same

File: src/controllers/dashboard/dlna/profile.js [hunk 8]
```diff
@@ -86,7 +87,9 @@ define(['jQuery', 'loading', 'globalize', 'emby-select', 'emby-button', 'emby-in
             index++;
             return li;
         }).join('') + '</div>';
-        var elem = $('.httpHeaderIdentificationList', page).html(html).trigger('create');
+        var elem = $('.httpHeaderIdentificationList', page);
```
Comment (ferferga) [inline src/controllers/dashboard/dlna/profile.js]: Same

File: src/controllers/dashboard/dlna/profile.js [hunk 9]
```diff
@@ -139,7 +142,9 @@ define(['jQuery', 'loading', 'globalize', 'emby-select', 'emby-button', 'emby-in
             li += '<button type="button" is="paper-icon-button-light" class="btnDeleteXmlAttribute listItemButton" data-index="0"><span class="material-icons delete"></span></button>';
             return li += '</div>';
         }).join('') + '</div>';
-        var elem = $('.xmlDocumentAttributeList', page).html(html).trigger('create');
+        var elem = $('.xmlDocumentAttributeList', page);
```
Comment (ferferga) [inline src/controllers/dashboard/dlna/profile.js]: Same

File: src/controllers/dashboard/dlna/profile.js [hunk 10]
```diff
@@ -183,7 +188,9 @@ define(['jQuery', 'loading', 'globalize', 'emby-select', 'emby-button', 'emby-in
             index++;
             return li;
         }).join('') + '</div>';
-        var elem = $('.subtitleProfileList', page).html(html).trigger('create');
+        var elem = $('.subtitleProfileList', page);
```
Comment (ferferga) [inline src/controllers/dashboard/dlna/profile.js]: Same

File: src/controllers/dashboard/dlna/profile.js [hunk 11]
```diff
@@ -275,7 +282,9 @@ define(['jQuery', 'loading', 'globalize', 'emby-select', 'emby-button', 'emby-in
         }
 
         html += '</ul>';
-        var elem = $('.directPlayProfiles', page).html(html).trigger('create');
+        var elem = $('.directPlayProfiles', page);
```
Comment (ferferga) [inline src/controllers/dashboard/dlna/profile.js]: Same

File: src/controllers/dashboard/dlna/profile.js [hunk 12]
```diff
@@ -336,7 +346,9 @@ define(['jQuery', 'loading', 'globalize', 'emby-select', 'emby-button', 'emby-in
         }
 
         html += '</ul>';
-        var elem = $('.transcodingProfiles', page).html(html).trigger('create');
+        var elem = $('.transcodingProfiles', page);
```
Comment (ferferga) [inline src/controllers/dashboard/dlna/profile.js]: Same

File: src/controllers/dashboard/dlna/profile.js [hunk 13]
```diff
@@ -370,15 +383,15 @@ define(['jQuery', 'loading', 'globalize', 'emby-select', 'emby-button', 'emby-in
...
+        currentSubProfile.AudioCodec = page.querySelector('#txtTranscodingAudioCodec').value;
+        currentSubProfile.VideoCodec = page.querySelector('#txtTranscodingVideoCodec').value;
+        currentSubProfile.Protocol = page.querySelector('#selectTranscodingProtocol').value;
         currentSubProfile.Context = 'Streaming';
-        currentSubProfile.EnableMpegtsM2TsMode = $('#chkEnableMpegtsM2TsMode', page).is(':checked');
-        currentSubProfile.EstimateContentLength = $('#chkEstimateContentLength', page).is(':checked');
-        currentSubProfile.TranscodeSeekInfo = $('#chkReportByteRangeRequests', page).is(':checked') ? 'Bytes' : 'Auto';
+        currentSubProfile.EnableMpegtsM2TsMode = $('#chkEnableMpegtsM2TsMode', page).matches(':checked');
+        currentSubProfile.EstimateContentLength = $('#chkEstimateContentLength', page).matches(':checked');
+        currentSubProfile.TranscodeSeekInfo = $('#chkReportByteRangeRequests', page).matches(':checked') ? 'Bytes' : 'Auto';
```
Comment (ferferga) [inline src/controllers/dashboard/dlna/profile.js:394]: Same in these three lines

File: src/controllers/dashboard/dlna/profile.js [hunk 14]
```diff
@@ -572,7 +591,9 @@ define(['jQuery', 'loading', 'globalize', 'emby-select', 'emby-button', 'emby-in
         }
 
         html += '</ul>';
-        var elem = $('.mediaProfiles', page).html(html).trigger('create');
+        var elem = $('.mediaProfiles', page);
```
Comment (ferferga) [inline src/controllers/dashboard/dlna/profile.js]: Same

File: src/controllers/dashboard/dlna/settings.js [hunk 15]
```diff
@@ -4,14 +4,16 @@ define(['jQuery', 'loading', 'libraryMenu', 'globalize'], function ($, loading,
...
-        $('#txtBlastInterval', page).val(config.BlastAliveMessageIntervalSeconds);
+        page.querySelector('#txtClientDiscoveryInterval').value = config.ClientDiscoveryIntervalSeconds;
+        page.querySelector('#chkEnableServer').checked = config.EnableServer;
+        page.querySelector('#chkBlastAliveMessages').checked = config.BlastAliveMessages;
+        page.querySelector('#txtBlastInterval').value = config.BlastAliveMessageIntervalSeconds;
         var usersHtml = users.map(function (u) {
             return '<option value="' + u.Id + '">' + u.Name + '</option>';
         }).join('');
-        $('#selectUser', page).html(usersHtml).val(config.DefaultUserId || '');
+        const elem = $('#selectUser', page);
```
Comment (ferferga) [inline src/controllers/dashboard/dlna/settings.js]: Same

File: src/controllers/dashboard/dlna/settings.js [hunk 16]
```diff
@@ -21,11 +23,11 @@ define(['jQuery', 'loading', 'libraryMenu', 'globalize'], function ($, loading,
...
             config.EnablePlayTo = form.querySelector('#chkEnablePlayTo').checked;
             config.EnableDebugLog = form.querySelector('#chkEnableDlnaDebugLogging').checked;
-            config.ClientDiscoveryIntervalSeconds = $('#txtClientDiscoveryInterval', form).val();
-            config.EnableServer = $('#chkEnableServer', form).is(':checked');
-            config.BlastAliveMessages = $('#chkBlastAliveMessages', form).is(':checked');
-            config.BlastAliveMessageIntervalSeconds = $('#txtBlastInterval', form).val();
-            config.DefaultUserId = $('#selectUser', form).val();
+            config.ClientDiscoveryIntervalSeconds = form.querySelector('#txtClientDiscoveryInterval').value;
+            config.EnableServer = $('#chkEnableServer', form).matches(':checked');
+            config.BlastAliveMessages = $('#chkBlastAliveMessages', form).matches(':checked');
```
Comment (ferferga) [inline src/controllers/dashboard/dlna/settings.js:28]: Same in this section

File: src/controllers/dashboard/general.js [hunk 17]
```diff
@@ -4,11 +4,13 @@ define(['jQuery', 'loading', 'globalize', 'emby-checkbox', 'emby-textarea', 'emb
     function loadPage(page, config, languageOptions, systemInfo) {
         page.querySelector('#txtServerName').value = systemInfo.ServerName;
         page.querySelector('#txtCachePath').value = systemInfo.CachePath || '';
-        $('#txtMetadataPath', page).val(systemInfo.InternalMetadataPath || '');
-        $('#txtMetadataNetworkPath', page).val(systemInfo.MetadataNetworkPath || '');
-        $('#selectLocalizationLanguage', page).html(languageOptions.map(function (language) {
+        page.querySelector('#txtMetadataPath').value = systemInfo.InternalMetadataPath || '';
+        page.querySelector('#txtMetadataNetworkPath').value = systemInfo.MetadataNetworkPath || '';
+        const elem = $('#selectLocalizationLanguage', page);
```
Comment (ferferga) [inline src/controllers/dashboard/general.js]: Same

File: src/controllers/dashboard/notifications/notification.js [hunk 18]
```diff
@@ -31,12 +31,12 @@ define(['jQuery', 'emby-checkbox'], function ($) {
...
             if (typeInfo.IsBasedOnUserEvent) {
-                $('.monitorUsers', page).show();
+                page.querySelector('.monitorUsers').classList.remove('hide');
             } else {
-                $('.monitorUsers', page).hide();
+                page.querySelector('.monitorUsers').classList.add('hide');
             }
 
-            $('.notificationType', page).html(typeInfo.Name || 'Unknown Notification');
+            $('.notificationType', page).innerHtml = typeInfo.Name || 'Unknown Notification';
```
Comment (ferferga) [inline src/controllers/dashboard/notifications/notification.js:40]: Same

File: src/controllers/dashboard/plugins/add.js [hunk 19]
```diff
@@ -42,24 +45,30 @@ define(['jQuery', 'loading', 'libraryMenu', 'globalize', 'connectionManager', 'e
...
+            overview.innerHtml = pkg.overview;
         } else {
-            $('#overview', page).hide();
+            page.querySelector('#overview').classList.add('hide');
         }
 
-        $('#description', page).html(pkg.description);
-        $('#developer', page).html(pkg.owner);
+        $('#description', page).innerHtml = pkg.description;
+        $('#developer', page).innerHtml = pkg.owner;
```
Comment (ferferga) [inline src/controllers/dashboard/plugins/add.js:61]: Same, here nothing changes in fact

File: src/controllers/dashboard/users/usernew.js [hunk 20]
```diff
@@ -27,20 +29,23 @@ define(['jQuery', 'loading', 'globalize', 'emby-checkbox'], function ($, loading
         }
 
         html += '</div>';
-        $('.channelAccess', page).show().html(html).trigger('create');
+        const channelAccess = $('.channelAccess', page);
```
Comment (ferferga) [inline src/controllers/dashboard/users/usernew.js:32]: Same

File: src/elements/emby-itemscontainer/emby-itemscontainer.js [hunk 21]
```diff
@@ -24,7 +24,6 @@ define(['itemShortcuts', 'inputManager', 'connectionManager', 'playbackManager',
     }
 
     function onContextMenu(e) {
-        var itemsContainer = this;
```
Comment (ferferga) [inline src/elements/emby-itemscontainer/emby-itemscontainer.js:27]: Why?

Comment (MrTimscampi) [reply src/elements/emby-itemscontainer/emby-itemscontainer.js:27]: Unused variable

File: src/components/tvproviders/schedulesdirect.js [hunk 22]
```diff
@@ -152,14 +154,20 @@ define(['jQuery', 'loading', 'globalize', 'emby-checkbox', 'listViewStyle', 'emb
...
                 info.EnableAllTuners = page.querySelector('.chkAllTuners').checked;
-                info.EnabledTuners = info.EnableAllTuners ? [] : $('.chkTuner', page).get().filter(function (i) {
-                    return i.checked;
-                }).map(function (i) {
-                    return i.getAttribute('data-id');
-                });
+
+                let enabledTuners = [];
+                if (!info.EnableAllTuners) {
+                    enabledTuners = Array.prototype.filter.call(page.querySelectorAll('.chkTuner'), (tuner) => {
```
Comment (masterkoppa) [inline src/components/tvproviders/schedulesdirect.js:163]: You can also leverage the spread operator to quickly convert them to an array to call `filter` on.

```suggestion
                    enabledTuners = [...page.querySelectorAll('.chkTuner')].filter((tuner) => {
```

Comment (MrTimscampi) [reply src/components/tvproviders/schedulesdirect.js:163]: Oh, good to know, thanks.

File: src/controllers/dashboard/general.js [hunk 23]
```diff
@@ -4,11 +4,13 @@ define(['jQuery', 'loading', 'globalize', 'emby-checkbox', 'emby-textarea', 'emb
...
-        $('#txtMetadataNetworkPath', page).val(systemInfo.MetadataNetworkPath || '');
-        $('#selectLocalizationLanguage', page).html(languageOptions.map(function (language) {
+        page.querySelector('#txtMetadataPath').value = systemInfo.InternalMetadataPath || '';
+        page.querySelector('#txtMetadataNetworkPath').value = systemInfo.MetadataNetworkPath || '';
+        const elem = page.querySelector('#selectLocalizationLanguage');
+        elem.innerHtml = languageOptions.map(function (language) {
             return '<option value="' + language.Value + '">' + language.Name + '</option>';
-        })).val(config.UICulture);
+        });
+        elem.val(config.UICulture);
```
Comment (grafixeyehero) [inline src/controllers/dashboard/general.js:13]: `elem.value = config.UICulture;`

File: src/controllers/dashboard/general.js [hunk 24]
```diff
@@ -4,11 +4,13 @@ define(['jQuery', 'loading', 'globalize', 'emby-checkbox', 'emby-textarea', 'emb
     function loadPage(page, config, languageOptions, systemInfo) {
         page.querySelector('#txtServerName').value = systemInfo.ServerName;
         page.querySelector('#txtCachePath').value = systemInfo.CachePath || '';
-        $('#txtMetadataPath', page).val(systemInfo.InternalMetadataPath || '');
-        $('#txtMetadataNetworkPath', page).val(systemInfo.MetadataNetworkPath || '');
-        $('#selectLocalizationLanguage', page).html(languageOptions.map(function (language) {
+        page.querySelector('#txtMetadataPath').value = systemInfo.InternalMetadataPath || '';
+        page.querySelector('#txtMetadataNetworkPath').value = systemInfo.MetadataNetworkPath || '';
+        const elem = page.querySelector('#selectLocalizationLanguage');
+        elem.innerHtml = languageOptions.map(function (language) {
```
Comment (grafixeyehero) [inline src/controllers/dashboard/general.js:10]: typo `elem.innerHTML `

File: src/components/tvproviders/schedulesdirect.js [hunk 25]
```diff
@@ -63,10 +63,12 @@ define(['jQuery', 'loading', 'globalize', 'emby-checkbox', 'listViewStyle', 'emb
...
                     return 0;
                 });
-                $('#selectCountry', page).html(countryList.map(function (c) {
+                const selectCountry = page.querySelector('#selectCountry');
+                selectCountry.innerHtml = countryList.map(function (c) {
                     return '<option value="' + c.value + '">' + c.name + '</option>';
-                }).join('')).val(info.Country || '');
-                $(page.querySelector('.txtZipCode')).trigger('change');
+                }).join('');
+                selectCountry.val(info.Country || '');
```
Comment (grafixeyehero) [inline src/components/tvproviders/schedulesdirect.js:70]: `selectCountry.value = info.Country || '';`

File: src/components/tvproviders/schedulesdirect.js [hunk 26]
```diff
@@ -63,10 +63,12 @@ define(['jQuery', 'loading', 'globalize', 'emby-checkbox', 'listViewStyle', 'emb
 
                     return 0;
                 });
-                $('#selectCountry', page).html(countryList.map(function (c) {
+                const selectCountry = page.querySelector('#selectCountry');
+                selectCountry.innerHtml = countryList.map(function (c) {
```
Comment (grafixeyehero) [inline src/components/tvproviders/schedulesdirect.js:67]: typo `selectCountry.innerHTML`

File: src/components/tvproviders/schedulesdirect.js [hunk 27]
```diff
@@ -195,16 +204,16 @@ define(['jQuery', 'loading', 'globalize', 'emby-checkbox', 'listViewStyle', 'emb
...
             }).then(function (result) {
-                $('#selectListing', page).html(result.map(function (o) {
+                page.querySelector('#selectListing').innerHtml = result.map(function (o) {
                     return '<option value="' + o.Id + '">' + o.Name + '</option>';
-                }));
+                });
 
                 if (listingsId) {
-                    $('#selectListing', page).val(listingsId);
+                    page.querySelector('#selectListing', page).value = listingsId;
```
Comment (grafixeyehero) [inline src/components/tvproviders/schedulesdirect.js:216]: `page.querySelector('#selectListing').value = listingsId;`

File: src/controllers/dashboard/dlna/settings.js [hunk 28]
```diff
@@ -4,14 +4,16 @@ define(['jQuery', 'loading', 'libraryMenu', 'globalize'], function ($, loading,
...
+        page.querySelector('#chkEnableServer').checked = config.EnableServer;
+        page.querySelector('#chkBlastAliveMessages').checked = config.BlastAliveMessages;
+        page.querySelector('#txtBlastInterval').value = config.BlastAliveMessageIntervalSeconds;
         var usersHtml = users.map(function (u) {
             return '<option value="' + u.Id + '">' + u.Name + '</option>';
         }).join('');
-        $('#selectUser', page).html(usersHtml).val(config.DefaultUserId || '');
+        const elem = page.querySelector('#selectUser');
+        elem.innerHtml = usersHtml;
+        elem.val(config.DefaultUserId || '');
```
Comment (grafixeyehero) [inline src/controllers/dashboard/dlna/settings.js:16]: elem.innerHTML= usersHtml;
elem.value = config.DefaultUserId || '';

File: src/controllers/dashboard/notifications/notification.js [hunk 29]
```diff
@@ -9,7 +9,8 @@ define(['jQuery', 'emby-checkbox'], function ($) {
             return '<label><input is="emby-checkbox" class="' + cssClass + '" type="checkbox" data-itemid="' + u.Id + '"' + checkedHtml + '/><span>' + u.Name + '</span></label>';
         }).join('');
         html += '</div>';
-        elem.html(html).trigger('create');
+        elem.innerHtml = html;
```
Comment (grafixeyehero) [inline src/controllers/dashboard/notifications/notification.js:12]: typo elem.innerHTML = html;

File: src/components/mediaLibraryCreator/mediaLibraryCreator.js [hunk 30]
```diff
@@ -52,9 +52,13 @@ define(['loading', 'dialogHelper', 'dom', 'jQuery', 'components/libraryoptionsed
     }
 
     function initEditor(page, collectionTypeOptions) {
-        $('#selectCollectionType', page).html(getCollectionTypeOptionsHtml(collectionTypeOptions)).val('').on('change', function () {
+        const selectCollectionType = page.querySelector('#selectCollectionType');
+        selectCollectionType.innerHTML = getCollectionTypeOptionsHtml(collectionTypeOptions);
+
+        selectCollectionType.value = '';
+        selectCollectionType.on('change', function () {
```
Comment (grafixeyehero) [inline src/components/mediaLibraryCreator/mediaLibraryCreator.js:59]:
```suggestion
        selectCollectionType.addEventListener('change', function () {
```

File: src/controllers/dashboard/users/userlibraryaccess.js [hunk 31]
```diff
@@ -39,15 +39,17 @@ define(['jQuery', 'loading', 'libraryMenu', 'globalize'], function ($, loading,
         }
 
         html += '</div>';
-        $('.channelAccess', page).show().html(html);
+        const channelAccess = page.querySelector('.channelAccess');
+        channelAccess.classList.remove('hide');
+        channelAccess.innerHtml = html;
```
Comment (grafixeyehero) [inline src/controllers/dashboard/users/userlibraryaccess.js:44]: same here

File: src/controllers/dashboard/users/useredit.js [hunk 32]
```diff
@@ -26,8 +26,10 @@ define(['jQuery', 'loading', 'libraryMenu', 'globalize'], function ($, loading,
                 html += '<label><input type="checkbox" is="emby-checkbox" class="chkFolder" data-id="' + folder.Id + '" ' + checkedAttribute + '><span>' + folder.Name + '</span></label>';
             }
 
-            $('.deleteAccess', page).html(html).trigger('create');
-            $('#chkEnableDeleteAllFolders', page).prop('checked', user.Policy.EnableContentDeletion);
+            const deleteAccess = page.querySelector('.deleteAccess');
+            deleteAccess.innerHtml = html;
```
Comment (grafixeyehero) [inline src/controllers/dashboard/users/useredit.js:30]: same here

File: src/controllers/dashboard/scheduledtasks/scheduledtask.js [hunk 33]
```diff
@@ -32,8 +32,8 @@ define(['jQuery', 'loading', 'datetime', 'dom', 'globalize', 'emby-input', 'emby
             });
         },
         loadScheduledTask: function (view, task) {
-            $('.taskName', view).html(task.Name);
-            $('#pTaskDescription', view).html(task.Description);
+            view.querySelector('.taskName').innerHtml = task.Name;
+            view.querySelector('#pTaskDescription').innerHtml = task.Description;
```
Comment (grafixeyehero) [inline src/controllers/dashboard/scheduledtasks/scheduledtask.js:36]:
```suggestion
            view.querySelector('.taskName').innerHTML = task.Name;
            view.querySelector('#pTaskDescription').innerHTML = task.Description;
```

File: src/controllers/dashboard/plugins/add.js [hunk 34]
```diff
@@ -72,7 +81,7 @@ define(['jQuery', 'loading', 'libraryMenu', 'globalize', 'connectionManager', 'e
     }
 
     function performInstallation(page, name, guid, version) {
-        var developer = $('#developer', page).html().toLowerCase();
+        var developer = page.querySelector('#developer').innerHtml.toLowerCase();
```
Comment (grafixeyehero) [inline src/controllers/dashboard/plugins/add.js:84]:
```suggestion
        var developer = page.querySelector('#developer').innerHTML.toLowerCase();
```

File: src/controllers/dashboard/plugins/add.js [hunk 35]
```diff
@@ -42,24 +45,30 @@ define(['jQuery', 'loading', 'libraryMenu', 'globalize', 'connectionManager', 'e
...
             var currentVersionText = globalize.translate('MessageYouHaveVersionInstalled', '<strong>' + installedPlugin.Version + '</strong>');
-            $('#pCurrentVersion', page).show().html(currentVersionText);
+            const currentVersion = page.querySelector('#pCurrentVersion');
+            currentVersion.classList.remove('hide');
+            currentVersion.innerHtml = currentVersionText;
         } else {
-            $('#pCurrentVersion', page).hide().html('');
+            const currentVersion = page.querySelector('#pCurrentVersion');
+            currentVersion.classList.add('hide');
+            currentVersion.innerHtml = '';
```
Comment (grafixeyehero) [inline src/controllers/dashboard/plugins/add.js:71]:
```suggestion
            currentVersion.innerHTML = '';
```

File: src/controllers/dashboard/plugins/add.js [hunk 36]
```diff
@@ -42,24 +45,30 @@ define(['jQuery', 'loading', 'libraryMenu', 'globalize', 'connectionManager', 'e
...
-        $('#developer', page).html(pkg.owner);
+        page.querySelector('#description').innerHtml = pkg.description;
+        page.querySelector('#developer').innerHtml = pkg.owner;
 
         if (installedPlugin) {
             var currentVersionText = globalize.translate('MessageYouHaveVersionInstalled', '<strong>' + installedPlugin.Version + '</strong>');
-            $('#pCurrentVersion', page).show().html(currentVersionText);
+            const currentVersion = page.querySelector('#pCurrentVersion');
+            currentVersion.classList.remove('hide');
+            currentVersion.innerHtml = currentVersionText;
```
Comment (grafixeyehero) [inline src/controllers/dashboard/plugins/add.js:67]:
```suggestion
            currentVersion.innerHTML = currentVersionText;
```

File: src/controllers/dashboard/plugins/add.js [hunk 37]
```diff
@@ -42,24 +45,30 @@ define(['jQuery', 'loading', 'libraryMenu', 'globalize', 'connectionManager', 'e
         populateVersions(pkg, page, installedPlugin);
         populateHistory(pkg, page);
 
-        $('.pluginName', page).html(pkg.name);
-        $('#btnInstallDiv', page).removeClass('hide');
-        $('#pSelectVersion', page).removeClass('hide');
+        page.querySelector('.pluginName').innerHtml = pkg.name;
```
Comment (grafixeyehero) [inline src/controllers/dashboard/plugins/add.js:48]:
```suggestion
        page.querySelector('.pluginName').innerHTML = pkg.name;
```

File: src/controllers/dashboard/plugins/add.js [hunk 38]
```diff
@@ -42,24 +45,30 @@ define(['jQuery', 'loading', 'libraryMenu', 'globalize', 'connectionManager', 'e
...
+        page.querySelector('#btnInstallDiv').classList.remove('hide');
+        page.querySelector('#pSelectVersion').classList.remove('hide');
 
         if (pkg.overview) {
-            $('#overview', page).show().html(pkg.overview);
+            const overview = page.querySelector('#overview');
+            overview.classList.remove('hide');
+            overview.innerHtml = pkg.overview;
```
Comment (grafixeyehero) [inline src/controllers/dashboard/plugins/add.js:55]:
```suggestion
            overview.innerHTML = pkg.overview;
```

File: src/controllers/dashboard/plugins/add.js [hunk 39]
```diff
@@ -42,24 +45,30 @@ define(['jQuery', 'loading', 'libraryMenu', 'globalize', 'connectionManager', 'e
...
         } else {
-            $('#overview', page).hide();
+            page.querySelector('#overview').classList.add('hide');
         }
 
-        $('#description', page).html(pkg.description);
-        $('#developer', page).html(pkg.owner);
+        page.querySelector('#description').innerHtml = pkg.description;
+        page.querySelector('#developer').innerHtml = pkg.owner;
```
Comment (grafixeyehero) [inline src/controllers/dashboard/plugins/add.js:61]:
```suggestion
        page.querySelector('#description').innerHTML = pkg.description;
        page.querySelector('#developer').innerHTML = pkg.owner;
```

File: src/controllers/dashboard/plugins/add.js [hunk 40]
```diff
@@ -11,7 +11,7 @@ define(['jQuery', 'loading', 'libraryMenu', 'globalize', 'connectionManager', 'e
             html += '<div style="margin-bottom:1.5em;">' + version.changelog + '</div>';
         }
 
-        $('#revisionHistory', page).html(html);
+        page.querySelector('#revisionHistory').innerHtml = html;
```
Comment (grafixeyehero) [inline src/controllers/dashboard/plugins/add.js:14]:
```suggestion
        page.querySelector('#revisionHistory').innerHTML = html;
```

File: src/controllers/dashboard/plugins/add.js [hunk 41]
```diff
@@ -22,10 +22,13 @@ define(['jQuery', 'loading', 'libraryMenu', 'globalize', 'connectionManager', 'e
             html += '<option value="' + version.version + '">' + version.version + '</option>';
         }
 
-        var selectmenu = $('#selectVersion', page).html(html);
+        var selectmenu = page.querySelector$('#selectVersion');
+        selectmenu.innerHtml = html;
```
Comment (grafixeyehero) [inline src/controllers/dashboard/plugins/add.js:26]:
```suggestion
        selectmenu.innerHTML = html;
```

File: src/controllers/dashboard/plugins/add.js [hunk 42]
```diff
@@ -22,10 +22,13 @@ define(['jQuery', 'loading', 'libraryMenu', 'globalize', 'connectionManager', 'e
...
 
         if (!installedPlugin) {
-            $('#pCurrentVersion', page).hide().html('');
+            const currentVersion = page.querySelector('#pCurrentVersion');
+            currentVersion.classList.add('hide');
+            currentVersion.innerHtml = '';
```
Comment (grafixeyehero) [inline src/controllers/dashboard/plugins/add.js:31]:
```suggestion
            currentVersion.innerHTML = '';
```

File: src/controllers/dashboard/notifications/notification.js [hunk 43]
```diff
@@ -31,12 +32,12 @@ define(['jQuery', 'emby-checkbox'], function ($) {
...
             if (typeInfo.IsBasedOnUserEvent) {
-                $('.monitorUsers', page).show();
+                page.querySelector('.monitorUsers').classList.remove('hide');
             } else {
-                $('.monitorUsers', page).hide();
+                page.querySelector('.monitorUsers').classList.add('hide');
             }
 
-            $('.notificationType', page).html(typeInfo.Name || 'Unknown Notification');
+            page.querySelector('.notificationType').innerHtml = typeInfo.Name || 'Unknown Notification';
```
Comment (grafixeyehero) [inline src/controllers/dashboard/notifications/notification.js:40]: same here

**Pull request closed without merging**